### DISCUSSION TITLE
Explore filter icon disappears

### DISCUSSION THREAD

**Discussion by include** [community/community#42949] [Mobile]
I've switched the explore filter to list only stars. Now I want to revert but the icon disappeared 😃 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.93.0 (85890385)
Device: iPhone 7 (iOS 15.7.2)
</details>

**Comment by HenSquared** [comment]
@include , Do you mind sharing a video of this behavior if you get a chance? 

I wasn't able to reproduce this.

**Reply by include** [reply]
Hi @HenSquared hwre it goes the link to the screen record.

https://drive.google.com/file/d/1DIW-QZBAros_mCjqYMRc2uhYiOpZ4IGW/view?usp=drivesdk

As you can see the small icon at your right (to select the filter) disappears :). I've never used this filter before and I was able to see the entire list without filtering anything, then on day I just changed this to search for some stuff in particular and then.. puff the app stated having this behavior :) 

let me know if you did find the glitch.

🙏

cheers,
Francisco

**Reply by HenSquared** [reply]
Hey @include 

Thanks a lot for the screen recording. This helps us a lot. I've opened up an issue internally to have this discussed by our team.
